### PR TITLE
What is your name?

### PR DESCRIPTION
**Pull request by Hanna888H**

### CODE REVIEW

File: Main.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import java.util.Scanner;
+    public class Main {
+        public static void main(String[] args) {
+            Scanner sc = new Scanner(System.in);
+            System.out.println("What is your name?");
+            String a  = sc.nextLine();
+            System.out.println("Your name is " + a);
+            System.out.println("What is your adress"+ ","+a +"?");
```
Comment (Hmeliumm) [inline Main.java]: Please format it with Ctrl+L

File: Main.java [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import java.util.Scanner;
+    public class Main {
```
Comment (Hmeliumm) [inline Main.java]: Please change to more clear name which will describe what class do

**Review by Hmeliumm — CHANGES_REQUESTED** [review]

File: Main.java (hunk 2 above)
Comment (Hanna888H) [reply Main.java]: Class name is changed to PersonalData from Main

File: Main.java (hunk 1 above)
Comment (Hanna888H) [reply Main.java]: Used code formatting Ctrl + Alt + L

Comment (Hmeliumm) [reply Main.java]: Got it

**Review by Hmeliumm — APPROVED** [review]

**Pull request merged**